uxarray/_exodus.py

```diff
@@ -141,19 +141,20 @@ def _read_exodus(ext_ds, ds_var_names):
     return ds
 
 
-def _write_exodus(ds, outfile, ds_var_names):
+def _encode_exodus(ds, outfile, ds_var_names):
```

Comment:
`outfile` does not need to be an argument anymore

Reply:
`outfile` is used in this function as a title for the dataset. Removing it doesn't change the overall funcitonality, however, I wasn't sure if removing it (and corresponding uses for the title) would be what was/is wanted and if the title in the dataset was required for following exodus standards or was just something we added